packages/react-chat/src/constants.ts

```diff
@@ -23,6 +23,7 @@ export enum ClassName {
   PROMPT = 'vfrc-prompt',
   SYSTEM_RESPONSE = 'vfrc-system-response',
   TIMESTAMP = 'vfrc-timestamp',
+  TEXTAREA = 'vfrc-textarea',
```

Comment:
this is unused, we can remove it. I'm fine with just keeping it as `id="vf-chat-input"` since people doing stylesheet mods already target it.
We want to keep out classnames and ids consistent as possible - otherwise we risk messing  up people's stuff in production.